### PR TITLE
Fixing System.Console tests for uapaot

### PR DESCRIPTION
**Pull request by joperezr**
cc: @danmosemsft @ianhays 

Fixing System.Console tests for uapaot.

### CODE REVIEW

File: src/System.Console/tests/Configurations.props [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uapaot-Windows_NT;
```
Comment (danmosemsft) [inline src/System.Console/tests/Configurations.props]: Why do we need a uap/uapaot configuration? This just segregates the Unix tests into another file, just like should have been done already and we do elsewhere already.

The configuration would be in the case we need #if uap or include files only when compiling for uap which is not the case here.

Comment (joperezr) [inline src/System.Console/tests/Configurations.props]: that is a good point, I'll create a netstandard-Windows_NT config then

**Comment by stephentoub** [comment]
LGTM, but why is this necessary?  Is it that the uap tests can't contain a Unix-specific DllImport?

**Comment by joperezr** [comment]
> Is it that the uap tests can't contain a Unix-specific DllImport?

Correct. In uapaot we ILC the runtime which contains the test assembly, and so if the test assembly has a PInvoke to a native library, the ILC compiled executable will have a dependency to that native library as well. Given that we don't support lazy assembly loading in an ILC executable, then we try to load this native library at startup which of course will fail since libc.dll won't be there.

**Comment by stephentoub** [comment]
Thanks, @joperezr.

**Pull request merged**